docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,192 @@
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Improve performance of reading from Parquet files with a large number of row
+  groups. ({issue}`24618`)
+* Improve performance of reading from large files on S3. ({issue}`24521`)
+
+## Iceberg connector
+
+* Add support for the `FIRST`, `AFTER`, and `LAST` clauses to `ALTER TABLE ...
+  ADD COLUMN`. ({issue}`20091`)
+* Add support for SSE-C in S3 security mapping. ({issue}`24566`)
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Add `$entries` metadata table. ({issue}`24172`)
+* Add `$all_entries` metadata table. ({issue}`24543`)
+* Allow configuring the `parquet_bloom_filter_columns` table property. ({issue}`24573`)
+* Allow configuring the `orc_bloom_filter_columns` table property. ({issue}`24584`)
+* Add the `rollback_to_snapshot` table procedure. The existing
+  `system.rollback_to_snapshot` procedure is deprecated. ({issue}`24580`)
+* Improve performance when listing columns. ({issue}`23909`)
+* Improve performance of reading from Parquet files with a large number of row
+  groups. ({issue}`24618`)
+* Improve performance of reading from large files on S3. ({issue}`24521`)
+* Remove the oldest tracked version metadata files when
+  `write.metadata.delete-after-commit.enabled` is set to `true`. ({issue}`19582`)
+* Correct catalog information in JMX metrics when using file system caching with
+  multiple catalogs. ({issue}`24510`)
+* Fix table read failures when using the Alluxio file system. ({issue}`23815`)
+* Prevent return of incomplete results by the `table_changes` table function. ({issue}`24709`) 
+* Prevent failures on queries accessing tables with multiple nested partition columns. ({issue}`24709`)
+
+## Ignite connector
+
+* Add support for `MERGE` statements. ({issue}`24443`)
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
+
+## Kudu connector
+
+* Add support for unpartitioned table. ({issue}`24661`)
+
+## MariaDB connector
+
+* Add support for the `FIRST`, `AFTER`, and `LAST` clauses to `ALTER TABLE ...
+  ADD COLUMN`. ({issue}`24735`)
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
+
+## MySQL connector
+
+* Add support for the `FIRST`, `AFTER`, and `LAST` clauses to `ALTER TABLE ...
+  ADD COLUMN`. ({issue}`24735`)
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
+
+## Oracle connector
+
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
+
+## Phoenix connector
+
+* Allow configuring scan page timeout with the
+  `phoenix.server-scan-page-timeout` configuration property. ({issue}`24689`)
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
+
+## PostgreSQL connector
+
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
+
+## Redshift connector
+
+* Improve performance of reading from Redshift tables by using the Redshift
```

Comment:
"by using the Redshift UNLOAD command" is an implementation detail. I'd remove it, or at most say "by increasing parallelism"

Reply:
I removed it..